influence_benchmark/RL/run_KTO_iteration.py

```diff
@@ -87,23 +91,20 @@ def format_dataset(example):
         num_positives * kto_config.desirable_weight / (num_negatives * kto_config.undesirable_weight),
     )
 
-    if args.lora_path is not None:
-        model.load_adapter(args.lora_path, adapter_name="adapter_to_train")
```

Comment:
Were we not using this anywhere?

Reply:
We were, but in order for LoRA adapters to play nice with FSDP, the model needs to be wrapped in a PeftModel. The easiest way to do that was to pass a `peft_config` to the `Trainer` instantiation. The trainable adapter is now `default`, while the reference adapter is the same `reference_adapter`.